source/index.html.md

```diff
@@ -436,6 +436,371 @@ Definition of the object attributes
 }
 ```
 
+## Organization Endpoint
+You can upload organization data to Shipamax. Organization data can be retrieved by ID, or by a query filter. You can also update an organization's details and delete an organization by ID.
+
+### POST
+Using the following endpoint you can create a new organization, you will need to provide a JSON body with all of the required details. Organizations are unique to their `externalIds`.
+
+https://public.shipamax-api.com/api/v2/Organizations
+
```

Comment:
This doesn't match the style of the other documentation in the same file. No need to include the domain name. The style of the API endpoint immediately before this one, for example, is:
| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /FileGroups/{id}/validationResult | POST  | Submit a new validationResult making it the lastValidationesult of the FileGroup  | 


The description of the endpoint should also focus on what it _means_, if that makes sense.

```suggestion
## Organization Endpoint
An Organization represents a business or other third-party that is referenced from the documents that Shipamax processes, like the shipper or consignee for a bill of lading. Each Organization must have a unique identifier provided by you, usually the identifier used in your own system, which we call `externalId` and is returned in details of processed documents. However, the `id` in the Organization API calls is the Shipamax specific ID.

| Endpoint                         | Verb  | Description                                                                       |
| -------------------------------- | ----- | --------------------------------------------------------------------------------- |
| /Organizations | GET | Retrieve a list of your Organizations that match a filter  | 
| /Organizations | POST  | Create a new Organization  | 
| /Organizations/{id} | PATCH  | Update an Organization by specifying the fields that have changed  | 
| /Organizations/{id} | DELETE | Remove an Organization from the system  | 
```
There are probably other inconsistencies too.